apps/mail/app/(auth)/login/login-client.tsx

```
@@ -169,24 +212,24 @@
                   {missingProviders.map((provider) => (
                     <div
                       key={provider.id}
-                      className="overflow-hidden rounded-md border border-black/10 dark:border-white/10"
+                      className="overflow-hidden rounded-md border border-white/10"
                     >
                       <button
                         onClick={() => toggleProvider(provider.id)}
-                        className="flex w-full items-center justify-between bg-black/5 px-4 py-3 text-left transition-colors hover:bg-black/10 dark:bg-white/5 dark:hover:bg-white/10"
+                        className="flex w-full items-center justify-between bg-white/5 px-4 py-3 text-left transition-colors hover:bg-white/10"
                       >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)